superset-frontend/src/views/CRUD/data/dataset/AddDataset/index.tsx

```diff
@@ -65,6 +65,9 @@ export function datasetReducer(
   }
 }
 
+const prevUrl =
```

Comment:
this is still temporary?

Reply:
I'm not sure the status of this URL, I just moved it out so it didn't get lost in the middle of the other components.